src/components/graph/menus/root-network-node-editor.tsx

```diff
@@ -176,24 +176,18 @@ const RootNetworkNodeEditor = () => {
 
     const rootNetworkModifiedNotification = useCallback(
         (event: MessageEvent<string>) => {
-            const parsedEventData: unknown = JSON.parse(event.data);
-            const eventData = parsedEventData as RootNetworkModifiedEventData;
-            const updateTypeHeader = eventData.headers.updateType;
-            if (
-                updateTypeHeader === NotificationType.ROOT_NETWORK_MODIFIED ||
-                updateTypeHeader === NotificationType.ROOT_NETWORK_UPDATED
```

Comment:
I seems we have two updateType for this eventData, so the updateType of this event data should be an union of NotificationType.ROOT_NETWORK_MODIFIED | NotificationType.ROOT_NETWORK_UPDATED